src/bp/ui-studio/src/web/views/FlowBuilder/inspector/index.tsx

```diff
@@ -128,8 +128,8 @@ class Inspector extends Component<Props> {
           node={currentFlowNode}
           updateNode={updateNodeAndRefresh}
           updateFlow={updateFlow}
-          copyFlowNodeElement={copyFlowNodeElement}
-          pasteFlowNodeElement={pasteFlowNodeElement}
+          copyFlowNodeElement={this.props.copyFlowNodeElement}
+          pasteFlowNodeElement={this.props.pasteFlowNodeElement}
```

Comment:
Good catch ! Actually, there is the same issue with SkillCallNode. Instead of adding `this.props` here, could you add the two methods in the deconstructor at the top of the method? `const { ... } = this.props`

Reply:
yeah, i will do that